### DISCUSSION TITLE
Why the "" Returns into &quot in Project Data while Saving it

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5901] [Q&A]
`"content": "{{completed_at | date: &quot;%Y-%m-%d&quot;}}",`
 
 This is what in project data but the expected content should be  `"content": "{{completed_at | date: "%Y-%m-%d"}}",`
 
 how can i prevent this ?

**Comment by RakulAgn** [comment]
@artf

**Comment by artf** [comment]
This shouldn't happen with the latest version, try to create a reproducible demo please

**Reply by RakulAgn** [reply]
I think the grapesjs mjml version is on 0.21.2 maybe that might be issue i think so let me check in latest version of app

**Reply by RakulAgn** [reply]
Yeah Got the issue in new version i just copied the text from slack and pasted  in the text block that `"content": "{{completed_at | date: &quot;%Y-%m-%d&quot;}}",` like this in project data. the copy paste brings this issue
